src/lang/en-US.ts

```diff
@@ -663,6 +669,34 @@ const Supabase: ResolvedIntlConfig['messages'] = {
     'supabase.poller.failed': `We encountered a problem retrieving the status of this action. Please check your network connection and try again.`,
 };
 
+const Legal: ResolvedIntlConfig['messages'] = {
+    'legal.heading': `Legal Stuff`,
```

Comment:
I imagine the content below is placeholder text that will be updated, if necessary, when Olivia reviews this PR. Leaving a comment behind to raise awareness.

Reply:
Somewhat... for legal that is approved by the legal team. Dave sent it and it has been approved.

The other stuff - yeah.